src/app/Services/alerts.service.ts

```diff
@@ -1,31 +1,34 @@
-import { Injectable } from '@angular/core';
+import { Injectable, Renderer2 } from '@angular/core';
 import { MatSnackBar } from '@angular/material';
+import { NGXLogger } from 'ngx-logger';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class AlertsService {
 
-  constructor(private _snackBar: MatSnackBar) { }
+  constructor(private _snackBar: MatSnackBar, private logger: NGXLogger) { }
 
   alertMessage(alertMessage: string, alertType: string) {
     var time;
     var dismiss;
     if (alertType == "warning" || alertType == "error") {
-      dismiss = ""
+      dismiss = "Got it!"
       time = 2000
     } else if (alertType == "success") {
-      dismiss = "";
+      dismiss = "Close";
       time = 3000;
     } else if (alertType == "info") {
       dismiss = "Got it!";
       time = 10000;
     }
-    this._snackBar.open(alertMessage, dismiss, {
+    
+    this.logger.info("Response alert with Info: '" + alertMessage + "', of type '" + alertType + ", with a duration of " + time + " milliseconds");
```

Comment:
We should use string interpolation when we concat strings, example:
 this.logger.info(`Response alert with Info: ${alertMessage}, of type${alertType}, with a duration of${time}milliseconds`);

Reply:
ok!

Reply:
Done